packages/block-library/src/navigation/edit.js

```diff
@@ -78,6 +78,11 @@ function Navigation( {
 		clientId
 	);
 
+	const placeholder = useRef();
+	useEffect( () => {
+		placeholder.current = <PlaceholderPreview />;
+	}, [] );
```

Comment:
or this preferable?
```jsx
const placeholder = useMemo( () => ( <PlaceholderPreview /> ) , [] );
```

Reply:
The is a third option in 

```
import { memo } from 'react';

const placeholder = memo( <Placeholder /> );
```

It might be worth it to wrap `placeholder-preview.js` itself with `memo`, as it is a display component that will never change between re-renders.

`export default memo( PlaceholderPreview );`

Reply:
I think using one of the memo functions will make it easier to scan and understand.

Reply:
Good suggestion @vcanales! However while testing it looks like the HoC memo doesn't appear to work this this context. I'll try the useMemo line next.
<img width="1324" alt="Screen Shot 2021-06-01 at 9 50 55 AM" src="https://user-images.githubusercontent.com/1270189/120361675-f9798e80-c2be-11eb-98ef-666153911466.png">

Reply:
Nice it looks like `useMemo` does the trick too
<img width="1330" alt="Screen Shot 2021-06-01 at 9 58 28 AM" src="https://user-images.githubusercontent.com/1270189/120362490-fc28b380-c2bf-11eb-926d-744540f0d46f.png">

Verified that there's a different top reason for re-renders ✨ Thanks @Mamaduka and @vcanales